README.md

```diff
@@ -25,15 +25,22 @@ instrument_server -k <API_KEY> -H <HOSTNAME>
 To start instrument_server as a daemon:
 
 ```
-instrument_server -k <API_KEY> -d
+instrument_server -k <API_KEY> start
```

Comment:
It's stupid, but this is a breaking change and maybe worth a major version change?